chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -105,8 +106,9 @@ func New(node *neutrino.ChainService, spendHintCache chainntnfs.SpendHintCache,
 
 		rescanErr: make(chan error),
 
-		chainUpdates: queue.NewConcurrentQueue(10),
-		txUpdates:    queue.NewConcurrentQueue(10),
+		chainUpdates: make(chan *filteredBlock, 100),
```

Comment:
Nit: we could also extract the `100` value into a parameter so that we can set it for production or testing.

Reply:
I don't see the point of this since I don't think setting it to certain values will let us test the race condition